src/configuration.js

```diff
@@ -31,47 +31,8 @@ import LoggingReporter from './reporters/logging_reporter';
 import RemoteSampler from './samplers/remote_sampler';
 import Tracer from './tracer';
 import UDPSender from './reporters/udp_sender';
-import {Validator} from 'jsonschema';
 import opentracing from 'opentracing';
 
-let configSchema = {
-    'type': 'object',
-    'required': ['jaeger'],
-    'properties': {
-        'jaeger': {
-            'type': 'object',
-            '$ref': '/jaeger'
-        }
-    },
-};
-
-let jaegerSchema = {
```

Comment:
(a) it provides an authoritative schema for the config (not just a file, but the actual data structure used in the code)
(b) it lets users do validation if they choose to